tests/webrtc.rs

```diff
@@ -0,0 +1,80 @@
+// Copyright 2023 litep2p developers
+//
+// Permission is hereby granted, free of charge, to any person obtaining a
+// copy of this software and associated documentation files (the "Software"),
+// to deal in the Software without restriction, including without limitation
+// the rights to use, copy, modify, merge, publish, distribute, sublicense,
+// and/or sell copies of the Software, and to permit persons to whom the
+// Software is furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+// OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+// FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+// DEALINGS IN THE SOFTWARE.
+
+use futures::StreamExt;
+use litep2p::{
+    config::ConfigBuilder as Litep2pConfigBuilder,
+    crypto::ed25519::Keypair,
+    protocol::{libp2p::ping, notification::ConfigBuilder},
+    transport::webrtc::config::Config,
+    types::protocol::ProtocolName,
+    Litep2p,
+};
+
+#[tokio::test]
+#[ignore]
+async fn webrtc_test() {
+    let _ = tracing_subscriber::fmt()
+        .with_env_filter(tracing_subscriber::EnvFilter::from_default_env())
+        .try_init();
+
+    let (ping_config, mut ping_event_stream) = ping::Config::default();
+    let (notif_config, mut notif_event_stream) = ConfigBuilder::new(ProtocolName::from(
+        // "/c0c89622f83f6f3c6b94bc307fec1652a3aa58ac88a564c34706633f44cbb3d1/block-announces/1",
```

Comment:
I think we can safely remove this

```suggestion
```

Reply:
May be just add the chain names and keep the commented-out genesis hash? the test is intended for manual invocation, so the protocol names are expected to be manually modified.

Reply:
I have no idea what chain the commented-out genesis hash belongs to. It's neither polkadot/kusama/westend/rococo/wococo. Let's delete it.